src/modules/extra/m_ldap.cpp

```diff
@@ -26,13 +26,21 @@
 #include "inspircd.h"
 #include "modules/ldap.h"
 
+#ifdef __GNUC__
```

Comment:
This can be inside the __APPLE__ ifdef.

Reply:
Resolved in 0a1262828d39dd31eaef41bb1fb4d21fa0965c84